WBooks/Library/LibraryController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  LibraryController.swift
+//  WBooks
+//
+//  Created by Nacho Mendez on 04/08/2021.
+//
+
+import UIKit
+
+class LibraryController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private lazy var library: LibraryView = LibraryView()
+    
+    override func loadView() {
+        view = library
+    }
+    
+    private let libraryViewModel = LibraryViewModel()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // NavigationBar
+        configurationNavigationBar()
+        
+        // delegates and data source
+        library.tableBooks.delegate = self
+        library.tableBooks.dataSource = self
+        
+        // cell .xib
+        let nib = UINib.init(nibName: "MyCustomCell", bundle: nil)
+        library.tableBooks.register(nib, forCellReuseIdentifier: "MyCustomCell")
+    }
+    
+    // func for the NavigationBar config
```

Comment:
Instead of comments to divide sections, we can use pragma marks. For example: 
```Swift
// MARK: - Navigation Bar
```
This marks are detected by Xcode and are the standard way to divide code in iOS.